TrackingTools/TrajectoryState/interface/ftsFromVertexToPoint.h

```diff
@@ -1,9 +1,7 @@
-#ifndef FTSFROMVERTEXTOPOINTFACTORY_H
-#define FTSFROMVERTEXTOPOINTFACTORY_H
+#ifndef TrackingTools_TrajectoryState_ftsFromVertexToPoint_h
+#define TrackingTools_TrajectoryState_ftsFromVertexToPoint_h
 // -*- C++ -*-
 //
-// Package:    EgammaElectronAlgos
-// Class:      FTSFromVertexToPointFactory
 //
 /**
  *  Generates a FreeTrajectoryState from a given measured point, vertex
```

Comment:
The description should be updated to reflect your last change.
Also add your name as "Modified by" after L16

Reply:
Which part exactly should I update, you want me to put this package tag back in with the new location? Or is it fine leaving it out and I should just update the description text with the example? Thanks

Reply:
I was thinking just at the example, when it says
```
 *  FTSFromVertexToPointFactory myFTS; myFTS(xmeas, xvert, momentum, charge);
 *  gives a FreeTrajectoryState of a track which comes from xvert to xmeas. 
```
Everything else seems still up to date to me